CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Patch
 
+- Masonry: Small cosmetic refactoring and fixed a tiny edge-case bug with `scrollContainer` prop (#471)
```

Comment:
should this go under `minor`?

Reply:
I guess the bug fix portion pushes it over. The cosmetic definitely wouldn't.